proposals/lip-0057.md

```diff
@@ -879,21 +881,22 @@ The definition and rationale for the `isCertificateGenerated` function is part o
 
 ```python
```

Comment:
Do we now still need the following sentence above:

> It's logic is the concatenation of the functions hasWaited and isPunished that are rationalized in [LIP 0023](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0023.md#explicit-unlock-mechanism) and [LIP 0024](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0024.md#rationale) respectively.

This seems not relevant at all for the specifications here and is rather confusing I find.

Reply:
also there is a typo :)

Reply:
We already refer to those LIPs in the rationale, so I think it is fine to remove it here. Can be resolved from my side.